### ISSUE TITLE
Row focus outline not visible when scrolling to right

### ISSUE THREAD

**Issue by tepi** [vaadin/components#113]
To reproduce:

1. Focus any row
2. Scroll the grid contents to the right
3. Focus indicator scrolls out of view; no way to tell which row is focused

**Comment by tomivirkki** [comment]
Fixed in 023473187744c5a53c85a6c403773a966d2a3a37
